doc/Overview/Mvux/Advanced/Commands.md

```diff
@@ -0,0 +1,405 @@
+---
+uid: Overview.Mvux.Advanced.Commands
+---
+
+# Commands
+
+This page covers the following topics:
+
+- [**What are commands**](#what-are-commands) - a brief overview of Commanding
+- [**Asynchronous commands**](#asynchronous-commands) - an overview of `IAsyncCommand`, MVUX's command that supports asynchronous execution.
+- Implicit command generation
+  - [**Basic Commands**](#basic-commands) - explains how commands are implicitly generated for methods declared in the Model.
+  - [**Using the CommandParameter**](#using-the-commandparameter) - how to use `CommandParameter`s.
+  - [**Additional Feed parameters**](#additional-feed-parameters) - demonstrates how Feeds can be consumed as parameters of the command method.
+  - [**Command generation rules**](#command-generation-rules) - a recap of implicit command generation rules.
+  - [**Configuring command generation using attributes**](#configuring-command-generation-using-attributes) - discusses ways to enable or disable implicit command generation and Feed parameters using attributes.
+- [**Manual command creation using factory methods**](#manual-command-creation-using-factory-methods) - explains another approach of creating commands explicitly
+- [**Cancelling a command**](#cancelling-a-command) -
```

Comment:
Does this section on cancelling a command exist?

Reply:
No, it should be removed. I added an examplanation based on David's description that commands are cancelled upon disposal of Model.